org/allPRs.ts

```diff
@@ -54,11 +54,66 @@ export const newColors = async () => {
     }
 }
 
+async function extractUrl(filePath: string, regex: string, matchGroup: any): Promise<string> {
+    const fileContents = await danger.github.utils.fileContents(filePath);
+    var fileMatch = fileContents.match(regex);
+    var extractedUrl = '';
+    if (Array.isArray(fileMatch) && fileMatch.length > matchGroup) {
+        extractedUrl = fileMatch[matchGroup];
+    }
+
+    return extractedUrl;
+}
+
+export const trackerBlockingMismatch = async() => {
+    // Fail if Tracker Blocking config URL is different between code and script
+    var appConfigUrlProviderFilePath = ''; 
+    var updateEmbeddedFilePath = ''; 
+    var configUrlProviderRegex = '';
+    var updateEmbeddedRegex = '';
+
+    // Configure
+    const repo = danger.github.thisPR.repo;
+    const modifiedFiles = danger.git.modified_files;
+
+    switch (repo) {
+        case "iOS":
+            appConfigUrlProviderFilePath = 'Core/AppURLs.swift';
+            updateEmbeddedFilePath = 'scripts/update_embedded.sh';
+
+            configUrlProviderRegex = 'static let trackerDataSet = URL.*string:.*staticBase.*trackerblocking\/(.*)\".*';
+            updateEmbeddedRegex = 'performUpdate \'https://staticcdn.duckduckgo.com/trackerblocking/(.*)\' \".*';
+            break;
+        case "macos-browser":
+            appConfigUrlProviderFilePath = 'DuckDuckGo/AppDelegate/AppConfigurationURLProvider.swift';
+            updateEmbeddedFilePath = 'scripts/update_embedded.sh';
+
+            configUrlProviderRegex = 'case \.privacyConfiguration: return URL.string: \"(.*)\".*';
+            updateEmbeddedRegex = 'CONFIG_URL=\"(.*)\"';
+            break;
```

Comment:
@loremattei this check is supposed to verify Tracker Data Set URL, but for macOS it actually checks Privacy Config URL. We could probably check both on both platforms?

Overall I think I'm fine with this approach (where we have to assume specific format of source files), but I think it would make sense to add comments in iOS and macOS sources to state that this code is checked by Danger which assumes specific and in case of changes the Danger check should be updated accordingly. Wdyt?